src/main/java/org/sobotics/guttenberg/clients/Guttenberg.java

```diff
@@ -166,16 +166,16 @@ private void checkLastExecution() {
 			return;
 		
 		Instant now = Instant.now();
-		Instant lastSuccess = StatusUtils.lastSucceededExecutionStarted;
+		Instant lastSuccess = StatusUtils.lastExecutionFinished;
 		
-		//long difference = lastSuccess.getEpochSecond() - now.getEpochSecond();
+		long difference = now.getEpochSecond() - lastSuccess.getEpochSecond();
 		
-		Instant criticalDate = now.minus(15, ChronoUnit.MINUTES);
+		//Instant criticalDate = now.minus(15, ChronoUnit.MINUTES);
 		
-		if (criticalDate.isBefore(lastSuccess)) {
+		if (difference > 15*60) {
```

Comment:
I think you could also have `ChronoUnit.MINUTES.between(lastSuccess, now) > 15`.